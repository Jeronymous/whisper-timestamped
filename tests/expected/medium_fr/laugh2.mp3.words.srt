1
00:00:00,160 --> 00:00:00,600
Hehehe

