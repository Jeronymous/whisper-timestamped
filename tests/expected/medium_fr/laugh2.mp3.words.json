{
  "text": " Hehehe",
  "segments": [
    {
      "id": 0,
      "seek": 0,
      "start": 0.22,
      "end": 0.64,
      "text": " Hehehe",
      "tokens": [
        50364,
        634,
        23500,
        50398
      ],
      "temperature": 0.0,
      "avg_logprob": -1.2401897430419921,
      "compression_ratio": 0.5,
      "no_speech_prob": 0.3699853718280792,
      "confidence": 0.173,
      "words": [
        {
          "text": "Hehehe",
          "start": 0.22,
          "end": 0.64,
          "confidence": 0.173
        }
      ]
    }
  ],
  "language": "fr"
}